lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -3810,12 +3838,20 @@ namespace {
 class TFPartition : public SILFunctionTransform {
   bool isTest = false;
   TensorFunctionClassifier tfc;
+  // For accelerator-only functions, map the SIL host function names to the
+  // lowered TF graph artifacts.
+  DenseMap<StringRef, SerializedGraphFunction> graphFunctions;
+
 public:
   TFPartition(bool isTest) : isTest(isTest) {}
 
   /// The entry point to the transformation.
   void run() override {
     SILFunction *hostFn = getFunction();
+    if (isAcceleratorOnly(*hostFn)) {
```

Comment:
in markFunction(), should the BBArgument's for the entryblock (i.e. the parameters) be marked as move?

Reply:
There is another on-going thread in https://github.com/apple/swift/pull/17460 discussing this -- I'll follow up with you and Richard.